devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -245,7 +245,10 @@ private void askTheUserForMissingValues() throws MojoExecutionException {
                             .replace("_", ".") + ".HelloResource";
                     className = prompter.promptWithDefaultValue("Set the resource classname", defaultResourceName);
                     if (StringUtils.isBlank(path)) {
-                        path = prompter.promptWithDefaultValue("Set the resource path ", "/hello");
+                        String[] resourceClassName = StringUtils.splitByCharacterTypeCamelCase(
+                                className.substring(className.lastIndexOf(".")));
+                        path = prompter.promptWithDefaultValue("Set the resource path ",
+                                "/" + resourceClassName[1].toLowerCase());
```

Comment:
Hmmm, the array is one based?

Reply:
array[0] is the point `.` :) , otherwise I can do `(className.lastIndexOf(".")+1)` when getting the substring.